ci-operator/templates/cluster-launch-installer-e2e.yaml

```diff
@@ -247,6 +250,16 @@ objects:
         fi
         cp $KUBECONFIG /tmp/admin.kubeconfig
         echo "Installation successful"
+        echo "Starting installer smoke tests..."
```

Comment:
Why are we running smoke tests in the e2e install path?  These should be two separate paths.  cluster-launch-installer-smoke or something.

Reply:
the smoke tests add about 5 seconds onto a successful e2e-aws run, checks that expected nodes are running/apiServer is up/all nodes,pods are healthy.. Should we create a separate cluster  to run the smoke tests? I was thinking they should be run with every PR, like e2e-aws.

Reply:
Because this job is going to be used across all repos, not just installer.  This is the job that every single PR on every single operator repo will use to deploy.  It's not appropriate to run installer specific tests on other people's PRs.

Reply:
ah, ok then got it - i'll separate the smoke tests in a different template